cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -80,6 +81,13 @@ type CMServer struct {
 	ServiceAccountKeyFile   string
 	RootCAFile              string
 
+	// volumeConfig
+	PersistentVolumeRecyclerDefaultScrubPod          string
+	PersistentVolumeRecyclerMinTimeoutNfs            int
```

Comment:
s/Nfs/NFS/

It would be nice not to hardcode the params for every plugin type here.  Why is this just NFS and HostPath?